src/payment/payment-strategy-type.ts

```diff
@@ -42,6 +42,7 @@ enum PaymentStrategyType {
     SEZZLE = 'sezzle',
     ZIP = 'zip',
     CONVERGE = 'converge',
+    MOLLIE = 'mollie',
```

Comment:
Can you please sort this options alphabetically 💅🏻

Reply:
You mean like all the gateway names?

Reply:
Correct